### ISSUE TITLE
Area buttons - reausable and exchangeable positions

### ISSUE THREAD

**Issue by dapgo** [enaon/ninebot-one-nRF52#4]
Idea/feature:
It would be good that code for  area "buttons" and code for linked functions are separated and just assigned, so it will make easier to replace code associated to a button or exchange positions.

And especially the EUC button and associated code, because users without a ninebot-one can reuse easily and keeping the code aligned with your original.

**Comment by enaon** [comment]
Yes, this is a nice idea. I haven't really finished the clock face, the memory button is there just for info during testing, it has no real reason to be there at all in a working watch. The alarm button can/will be moved to the minutes area, so that will be gone too. The euc button is speciefic, so that has no reason to be on the generic watch face as well. So yes, I will seperate this area so that it can be easier customized, thanks.

**Comment by dapgo** [comment]
I've created a simple [menu](https://github.com/dapgo/ninebot-one-nRF52/blob/master/p8-nb/my_agenda) page but  based in  coordinates from variables  (accessible from face and TouchHandler functions).  I add variables to handler file but I think variables should be loaded and unloaded when leaving the page to save RAM.
